teachertool/src/components/CatalogOverlay.tsx

```diff
@@ -106,34 +127,72 @@ const CatalogList: React.FC = () => {
         announceToScreenReader(lf("Added '{0}' to checklist.", getReadableCriteriaTemplate(c)));
     }
 
+    function getItemIdForTag(tag: string) {
+        return `accordion-item-${tag}`;
+    }
+
+    function onTagExpandToggled(tag: string, expanded: boolean) {
+        if (expanded) {
+            /* await */ addExandedCatalogTagAsync(tag);
```

Comment:
If we don't need these to be awaited, can we removed the commented `await`?

Reply:
Do we have a linter in this repo? We usually comment this or `async` to let the linter and the reader know that we explicitly and not accidentally running this in the background.

Reply:
This is a pattern we followed in other areas of the code to make it clear that we're intentionally *not* awaiting even though it's async (as opposed to an oversight). I believe it was initially a recommendation by @eanders-ms in another PR. I'm fine to remove or keep either way, but I wanted to be consistent.

Reply:
Gotcha! Yeah, I'm on board to keep it in to be consistent.